stack/stack.h

```diff
@@ -0,0 +1,21 @@
+#ifndef STACK_STACK_H
+#define STACK_STACK_H
+#include <stdbool.h>
+#include <stdlib.h>
+
+typedef struct Node {
+    int value;
+    struct Node *next;
+} Node;
```

Comment:
```suggestion
typedef struct Node Node;
```
Стек должен быть абстрактным типом данных. Стало быть внутренне представление стека должно быть скрыто от пользователя. А сам тип должен быть в .c-файле.